Dockerfile

```diff
@@ -18,6 +19,15 @@ RUN poetry install
 COPY wkconnect wkconnect
 RUN poetry install
 
+COPY fast_wkw fast_wkw
+RUN curl --output wkw.zip https://codeload.github.com/scalableminds/webknossos-wrap/zip/refs/heads/lz4-crate && \
+    unzip wkw.zip && \
+    cd fast_wkw && \
+    $HOME/.cargo/bin/cargo build --release && \
+    cp target/release/libfast_wkw*.so ../wkconnect/fast_wkw.so && \
+    cd .. && \
+    rm -r fast_wkw
```

Comment:
Does any of this have to happen for a local non-docker installation too? If so, could you adapt the readme? It currently only says poetry install

Reply:
Also I noticed this is duplicated in Dockerfile and the CI yaml. Maybe it could be extracted into a script?

Reply:
This will be removed once scalableminds/webknossos-wrap#66 is merged. Then, the regular rust package manager will load wkwrap without any special code.